pkgs/top-level/all-packages.nix

```diff
@@ -9772,6 +9772,10 @@ with pkgs;
     go = buildPackages.go_1_25;
   };
 
+  buildGccGoModule = callPackage ../build-support/go/module.nix {
+    go = buildPackages.gccgo;
+  };
+
```

Comment:
Why this instead of defining `go = gccgo` globally if Go isn't available? That's what we do for other packages, e.g. udev.  That way there's at least a chance things will work out of the box.

Reply:
I want to first add a way of using a gccgo-backed `buildGoModule`, so it's not necessary to get / emulate a POWER machine to test how functional this is. Switching the default `go` and `buildGoModule` attributes was gonna be the next step.

Reply:
Okay, makes sense!

Reply:
Alrighty, will drop for now.